COPYRIGHT.md

```diff
@@ -0,0 +1,206 @@
+# Copyright Contributors to the Kokkos Project
+
+This file contains a (partial) list of people who contributed to the Kokkos Project
+and retain applicable copyrights to their contributions.
+
+Contributors are listed by organization where applicable, and the email addresses
+under which they contributed are listed.
```

Comment:
"For each contributor, we list their organization (if known) and the email address associated with their commits." ?

Reply:
Done